grammar.js

```diff
@@ -1086,10 +1086,10 @@ module.exports = grammar({
       seq('unchecked', '(', $._expression, ')')
     ),
 
-    conditional_access_expression: $ => prec.right(seq(
+    conditional_access_expression: $ => prec.right(PREC.COND, seq(
       field('condition', $._expression),
       '?',
-      field('value', $._expression)
+      choice($.member_binding_expression, $.element_binding_expression)
```

Comment:
Are `member_binding_expression` and `element_binding_expression` used anywhere else still? If I understand them correctly, I think that they can *only* appear here in this conditional access expression.

Reply:
They're both still listed under options in `expression` as per the Roslyn syntax.

We need element_binding_expression there for syntax like

```csharp
var x = new Dictionary<string, int> { ["a"] = 65 };
```

I believe `member_binding_expression` can be removed from `expression` although it does cause the number of file regressions in Roslyn to go up by 1 but that is purely because of the known restriction we currently have with `await` not being able to be used as an identifier and the line

```csharp
isDynamic = await.AwaitableInfo.IsDynamic;
```

Parsing as a side-effect of allowing unqualified access to a member (it's treating it as `await .AwaitableInfo.IsDynamic`

Reply:
I've pushed an update that removes `member_binding_expression` from `expression`.